.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report
+about: Did you find a bug in PMDK? Please let us know.
+labels: "Type: Bug"
+---
+<!--
+Before creating new issue, ensure that similar issue wasn't already created
+  * Search: https://github.com/pmem/issues/issues
+
+Note that if you do not provide enough information to reproduce the issue, we may not be able to take action on your report.
+Remember this is just a minimal template. You can extend it with data you think may be useful.
+-->
+
+# ISSUE: <!-- fill the title of issue -->
+
+## Environment Information 
+ 
+- PMDK package version(s): <!-- fill this out -->
+- OS(es) version(s):       <!-- fill this out -->
```

Comment:
Please add request to fill also:

```suggestion
- OS(es) version(s):       <!-- fill this out -->
- ndctl version(s):         <!-- fill this out -->
- kernel version(s):       <!-- fill this out -->
- compiler, libraries, packaging and other related tools version(s):       <!-- fill this out -->
```
it would help us to reproduce customer environment